emote/nn/curl.py

```diff
@@ -162,7 +162,7 @@ def __init__(
         use_per_image_mask_size: bool = False,
         use_fast_augment: bool = False,
         use_projection_layer: bool = True,
-        augment_anchor_and_pos: bool = True,  # disabling this saves some computation and doesn't seem to have any adverse effects.
+        augment_anchor_and_pos: bool = False,  # disabling this saves some computation and doesn't seem to have any adverse effects.
```

Comment:
Is this default necessary to change? The other ones are bugfixes or opt-in, this might change semantics where both options are valid.

Reply:
Not necessary for sure!